frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx

```diff
@@ -484,7 +484,7 @@ class Visualization extends React.PureComponent {
           ) : (
             <div
               data-card-key={getCardKey(series[0].card)}
-              className="flex flex-column full-height"
+              className="flex flex-column flex-full"
```

Comment:
this change broke internal scroll for object detail viz in the query builder and on dashboards 😢 

Before | After
--- | ---
![scrolldetail](https://user-images.githubusercontent.com/30528226/223497186-a8f6e74c-b0db-42bb-b255-a5426ff23076.gif) | ![noScroll](https://user-images.githubusercontent.com/30528226/223497194-e251d42e-06bb-468a-b2ca-74eebff70d65.gif)

Reply:
Well... That's not ideal :/

Reply:
Does adding back the `full-height` class bring back the scroll functionality?

Reply:
yeah, having both classes seems to work for this situation

![Screen Shot 2023-03-07 at 10 16 13 AM](https://user-images.githubusercontent.com/30528226/223498442-817aad99-2691-4434-b5da-b9805d8f39d1.png)

Reply:
Well damn. I think having both classes made cards not render properly

Reply:
oh wait. it fixes the scrollbar issue, but it cutz of the paginator control.

![Screen Shot 2023-03-07 at 10 18 22 AM](https://user-images.githubusercontent.com/30528226/223498870-c2202a6a-a031-4918-88ed-dc63013c5779.png)

Reply:
Might need to get a bit more creative with a fix then. @calherries actually had a solution where he moved the wrapper up a couple levels. Not sure if that approach would tick all our boxes or not.

https://github.com/metabase/metabase/compare/fix-vis-wrapper

Reply:
You could try that approach, but the downside is the visualization header is included in the download png. I didn't check whether that was acceptable 
https://github.com/metabase/metabase/pull/28546

Reply:
We could also just make adjustments to the preview container

Reply:
I haven't looked at it as much as you, but the preview container sounds like the better place to look than messing with `Visualization` at all